add agread extra parameter

seems the parameter was always defined in prototype, just compiler was
not that picky about that

libtool: compile:  ccache x86_64-pld-linux-gcc -std=gnu99 -std=gnu99 -DHAVE_CONFIG_H -I. -I./config -I/usr/include/graphviz -I/usr/include/freetype2 -I/usr/include/libxml2 -I/usr/include/lqr-1 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -fopenmp -O2 -fwrapv -pipe -Wformat -Werror=format-security -gdwarf-4 -fno-debug-types-section -fvar-tracking-assignments -g2 -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4 -fPIC -march=x86-64 -gdwarf-4 -fno-debug-types-section -fvar-tracking-assignments -g2 -Wall -pthread -MT coders/coders_dot_la-dot.lo -MD -MP -MF coders/.deps/coders_dot_la-dot.Tpo -c coders/dot.c  -fPIC -DPIC -o coders/.libs/coders_dot_la-dot.o
coders/dot.c: In function 'ReadDOTImage':
coders/dot.c:138:3: error: too few arguments to function 'agread'
In file included from /usr/include/graphviz/types.h:49:0,
                 from /usr/include/graphviz/gvc.h:17,
                 from coders/dot.c:66:
/usr/include/graphviz/cgraph.h:268:18: note: declared here
make[1]: *** [coders/coders_dot_la-dot.lo] Error 1
make[1]: Leaving directory `/home/users/glen/rpm/BUILD/x86_64-linux/ImageMagick-6.7.5-9'

checked:
- auto/ti/graphviz-2_26_3-7
- auto/th/graphviz-2_20_3-9
- auto/ti/graphviz-2_18-4
- auto/ti/graphviz-2_14_1-6

--- ImageMagick-6.7.9-10/coders/dot.c~	2012-08-30 14:41:56.000000000 +0300
+++ ImageMagick-6.7.9-10/coders/dot.c	2013-03-20 00:23:13.217244621 +0200
@@ -136,7 +136,7 @@
   (void) AcquireUniqueFilename(read_info->filename);
   (void) FormatLocaleString(command,MaxTextExtent,"-Tsvg -o%s %s",
     read_info->filename,image_info->filename);
-  graph=agread(GetBlobFileHandle(image));
+  graph=agread(GetBlobFileHandle(image), (Agdisc_t *) 0);
   if (graph == (graph_t *) NULL)
     return ((Image *) NULL);
   option=GetImageOption(image_info,"dot:layout-engine");
